source/glest_game/menu/menu_state_custom_game.cpp

```diff
@@ -4369,26 +4369,29 @@ namespace Glest
 
           for (int i = 0; i < GameConstants::maxPlayers; ++i)
           {
-            if (i >= mapInfo.hardMaxPlayers && checkBoxAllowObservers.getValue() == false)
+            if (i >= mapInfo.hardMaxPlayers)
```

Comment:
I only refactored the if/else statement here. The stuff inside the block hasn't changed.